src/components/App/App.component.jsx

```diff
@@ -57,6 +42,18 @@ function App() {
               <Route exact path="/login">
                 <LoginPage />
               </Route>
+              <ProtectedRoute
+                path="/favorites"
+                isAuthenticated={globalState.authenticated}
+                Component={Favorites}
+              />
+              <DetailContextProvider>
+                <ProtectedRoute
+                  path="/favoriteDetail"
+                  isAuthenticated={globalState.authenticated}
+                  Component={FavoriteDetail}
+                />
```

Comment:
With the current routing implementation, the user won't be able to share links, because the routing depends on the storage. To handle correctly routing you should use parameters in the path: `/favorites/:id` and use `useParams` to obtain the id.
i.e.
```js
import { useParams } from 'react-router-dom';

function Component() {
  const { id } = useParams();
}
```